.gitignore

```diff
@@ -24,6 +24,9 @@ bld/
 [Oo]bj/
 [Ll]og/
 
+# Include cpprestsdk files in Release folder
+!src/cpprestsdk/cpprestsdk/Release
```

Comment:
Remove, This will prevent us from updating cpprestsdk in the future

Reply:
removed